backend/dataall/core/environment/services/environment_service.py

```diff
@@ -582,106 +566,42 @@ def list_valid_user_environments(session, data=None) -> dict:
-                )
-            )
-        return query
-
     @staticmethod
     def paginated_user_groups(session, data=None) -> dict:
         context = get_context()
         return paginate(
-            query=EnvironmentService.query_user_groups(session, context.username, context.groups, data),
+            query=EnvironmentRepository.query_user_groups(session, context.username, context.groups, data),
             page=data.get('page', 1),
             page_size=data.get('pageSize', 5),
         ).to_dict()
 
-    @staticmethod
-    def query_user_consumption_roles(session, username, groups, filter) -> Query:
-        query = (
-            session.query(ConsumptionRole)
-            .filter(ConsumptionRole.groupUri.in_(groups))
-            .distinct(ConsumptionRole.consumptionRoleName)
-        )
-        if filter and filter.get('term'):
-            term = filter['term']
-            query = query.filter(
-                or_(
-                    ConsumptionRole.consumptionRoleName.ilike('%' + term + '%'),
-                )
-            )
-        if filter and filter.get('groupUri'):
-            print('filter group')
-            group = filter['groupUri']
-            query = query.filter(
-                or_(
-                    ConsumptionRole.groupUri == group,
-                )
-            )
-        return query
-
     @staticmethod
     def paginated_user_consumption_roles(session, data=None) -> dict:
         context = get_context()
         return paginate(
-            query=EnvironmentService.query_user_consumption_roles(session, context.username, context.groups, data),
+            query=EnvironmentRepository.query_user_consumption_roles(session, context.username, context.groups, data),
             page=data.get('page', 1),
             page_size=data.get('pageSize', 5),
         ).to_dict()
 
-    @staticmethod
-    def query_user_environment_groups(session, groups, uri, filter) -> Query:
-        query = (
-            session.query(EnvironmentGroup)
-            .filter(EnvironmentGroup.environmentUri == uri)
-            .filter(EnvironmentGroup.groupUri.in_(groups))
-        )
-        if filter and filter.get('term'):
-            term = filter['term']
-            query = query.filter(
-                or_(
-                    EnvironmentGroup.groupUri.ilike('%' + term + '%'),
-                )
-            )
-        return query
-
     @staticmethod
     @ResourcePolicyService.has_resource_permission(environment_permissions.LIST_ENVIRONMENT_GROUPS)
     def paginated_user_environment_groups(session, uri, data=None) -> dict:
         return paginate(
-            query=EnvironmentService.query_user_environment_groups(session, get_context().groups, uri, data),
+            query=EnvironmentRepository.query_user_environment_groups(session, get_context().groups, uri, data),
             page=data.get('page', 1),
             page_size=data.get('pageSize', 1000),
         ).to_dict()
 
```

Comment:
Why not calling `query_all_environment_groups` directly? in the groupService and in the paginated_all_environment_groups?